tox.ini

```diff
@@ -0,0 +1,27 @@
+[flake8]
+max-line-length = 100
+
+[tox]
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/src:{toxinidir}/lib
```

Comment:
The `:{toxinidir}/lib` bit is unnecessary, as this doesn't use any charmcraft libs.